osmaxx/excerptexport/models/output_file.py

```diff
@@ -9,11 +9,18 @@
 from osmaxx.excerptexport.utils.upload_to import get_private_upload_storage
 
 
+def uuid_directory_path(instance, filename):
+    # file will be uploaded to MEDIA_ROOT/osmaxx/<public_uuid>/<filename>
+    return '{0}/{1}/{2}/{3}'.format('osmaxx', 'outputfiles', instance.public_identifier, filename)
```

Comment:
wouldn't `os.path.join('osmaxx', 'outputfiles', instance.public_identifier, filename)` be more appropriate?